Cargo.toml

```diff
@@ -6,7 +6,7 @@ members = [
 ]
 
 [workspace.package]
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Suggestion by @zie1ony , change to "0.2.0".

Reply:
Agree on this one, the rest is lgtm

Reply:
Done